contrib/muparser/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "muparser"
+pkgver = "2.3.4"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DENABLE_OPENMP=OFF"]
```

Comment:
this just needs libomp-devel in makedepends and then it works (most likely)

Reply:
Honestly, I really don't think a parsing library that is only used to implement a calculator in the `lxqt-runner` program needs multithreading...

If enabling it doesn't make things more fragile I guess it wouldn't hurt though.

Reply:
if they have it on by default then may as well leave it on

Reply:
done